test/battle/hold_effect/shell_bell.c

```diff
@@ -198,5 +198,95 @@ SINGLE_BATTLE_TEST("Shell Bell does not activate on Future Sight if the original
     }
 }
 
+DOUBLE_BATTLE_TEST("Shell Bell heals accumulated damage for spread moves")
+{
+    s16 opponentLeftDamage;
+    s16 opponentRightDamage;
+    s16 playerRightDamage;
+    s16 shellBellHeal;
+
+    const u16 maxHp = 200;
+    const u16 initHp = 1;
+    GIVEN {
```

Comment:
```suggestion
    GIVEN {
        ASSUME(gMovesInfo[MOVE_DISCHARGE].target == MOVE_TARGET_FOES_AND_ALLY);
```
Should check that discharge is an AoE move

Reply:
what's an AoE move?

Reply:
Area of Effect
Thing that hits everything on the field